doc/man3/SSL_set_connect_state.pod

```diff
@@ -39,11 +44,19 @@ using L<SSL_write_ex(3)>, L<SSL_write(3)>, L<SSL_read_ex(3)>, or L<SSL_read(3)>,
 the handshake routines must be explicitly set in advance using either
 SSL_set_connect_state() or SSL_set_accept_state().
 
+If SSL_is_server() is called before SSL_set_connect_state() or
+SSL_set_accept_state() is called (either automatically or explicitly),
+the result depends on what method is used when SSL_CTX is created with
+L<SSL_CTX_new(3)>. If a generic method or a dedicated server method is
+passed to L<SSL_CTX_new(3)>, SSL_is_server() returns 1 otherwise it returns 0.
```

Comment:
The concern here is with respect to the grammar nit of a "comma splice" (https://en.wikipedia.org/wiki/Comma_splice), and I don't at all insist that you spend time learning about it or even making a change to this text. (Which is why I said "I'd prefer" and not some stronger language.)  The short form is that if the stuff after the comma/semicolon could stand on its own as a complete sentence ("Otherwise, it returns 0."), then it is incorrect to use a comma, and a semicolon or full stop should be used.  ("Otherwise" should have a comma after it to offset the conjunctive adverb that starts the sentence.)  Hmm, though apparently the dictionary I have at hand is fine with the case that I would have called a comma splice, so definitely take this whole chain of comments as a side note.